edx_solutions_api_integration/courses/urls.py

```diff
@@ -58,6 +58,7 @@
     url(r'^{0}/updates/*$'.format(COURSE_ID_PATTERN), courses_views.CoursesUpdates.as_view()),
     url(r'^{0}/users/(?P<user_id>[0-9]+)$'.format(COURSE_ID_PATTERN), courses_views.CoursesUsersDetail.as_view()),
     url(r'^{0}/users/*$'.format(COURSE_ID_PATTERN), courses_views.CoursesUsersList.as_view()),
+    url(r'^{0}/engagement-summary/*$'.format(COURSE_ID_PATTERN), courses_views.CoursesEngagementSummary.as_view()),
```

Comment:
I think underscore `engagement_summary` is more intuitive in urls. `

Reply:
have changed it to `engagement_summary`